Makefile

```diff
@@ -15,26 +15,22 @@ help: ## Display help message
 	@perl -nle'print $& if m{^[\.a-zA-Z_-]+:.*?## .*$$}' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m  %-25s\033[0m %s\n", $$1, $$2}'
 
 build_and_run: ## Run and build application
-	sudo docker-compose up -d --build
+	 docker compose up -d --build
 
-run_app: ## Run application
-	sudo docker-compose up -d
+run_app:  ## Run application
+	docker compose up
```

Comment:
please add `-d` to up docker container as daemon process

Reply:
I drop `-d`, to get logs that are suitable for development, but I can add it back 🙏

Reply:
@vitayushchyk better to add, cause in any case we can check the logs in the, for instance, docker service in Pycharm